src/quo2/components/inputs/address_input/view.cljs

```diff
@@ -0,0 +1,160 @@
+(ns quo2.components.inputs.address-input.view
+  (:require [react-native.core :as rn]
+            [react-native.clipboard :as clipboard]
+            [quo2.theme :as quo.theme]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.icon :as icon]
+            [quo2.components.buttons.button.view :as button]
+            [quo2.components.inputs.address-input.style :as style]
+            [utils.i18n :as i18n]
+            [reagent.core :as reagent]))
+
+(defn- icon-color
+  [blur? theme]
+  (if blur?
+    (colors/theme-colors colors/neutral-80-opa-30 colors/white-opa-10 theme)
+    (colors/theme-colors colors/neutral-40 colors/neutral-60 theme)))
+
+(defn- clear-button
+  [{:keys [on-press blur? theme]}]
+  [rn/touchable-opacity
+   {:accessibility-label :clear-button
+    :style               style/clear-icon-container
+    :on-press            on-press}
+   [icon/icon :i/clear
+    {:color (icon-color blur? theme)
+     :size  20}]])
+
+(defn- loading-icon
+  [blur? theme]
+  [rn/view {:style style/clear-icon-container}
+   [icon/icon :i/loading
+    {:color (icon-color blur? theme)
+     :size  20}]])
+
+(defn- positive-state-icon
+  [theme]
+  [rn/view {:style style/clear-icon-container}
+   [icon/icon :i/positive-state
+    {:color (colors/theme-colors colors/success-50 colors/success-60 theme)
```

Comment:
nit, imo we don't want to directly use colors/success-50(/60). Better imo to use:
(colors/theme-colors 
(colors/custom-color :success 50)
(colors/custom-color :success 60)
theme)
and we let the resolver mechanism handle the look up.

Reply:
Sure, I can change it, thanks for the suggestion. Doing a quick search I detected several places where we are making use of `colors/success-50` and `colors/success-60`, I don't have a strong opinion about it, but maybe we can define those colors as private so we can only access to them via `custom-color` fn and keys in `colors-map`.
Do we have any definition or discussion about this I could read about?

Reply:
Cool, I updated this implementation for now, we can follow up on another PR with a general refactor once we agree on these colors guidelines 💪